website/docs/r/scheduled_view.html.markdown

```diff
@@ -44,6 +45,7 @@ The following arguments are supported:
 The following attributes are exported:
 
 - `id` - The internal ID of the scheduled view.
+- `index_id` - The Index ID of the scheduled view. It remains unchanged during resource updates, and any manual modifications will be disregarded. While it’s not mandatory, we recommend to ignore this via `ignore_changes = [index_id]`.
```

Comment:
Now we have made this field as computed you can drop the "modification" and ignore_changes part

Reply:
Yeah I have just kept if for the fact that if someone explicitly tries to set `index_id = "000000ABCDE..." (some random value)` in the resource definition, then with the help of `ignore_changes` that will be ignored.